lib/tasks/zomato.rake

```diff
@@ -0,0 +1,21 @@
+require_relative '../zomato'
+
+namespace :zomato do
+  desc 'Load cuisines'
+  task :cuisines, [:city_id]  => :environment do |task, args|
+    Zomato.new(args[:city_id].to_i).createCuisines
+  end
+
+  desc 'Load Restaurants'
+  task :restaurants, [:city_id, :from, :to, :count, :review_count] => :environment do |task, args|
+    start = args[:from].to_i
+    to = args[:to].to_i
+    count = args[:count].to_i
+    review_count = args[:review_count].to_i
+    city_id = args[:city_id].to_i
+    while start < to
```

Comment:
you can also iterate on a [range with step](https://ruby-doc.org/core-2.2.0/Range.html#method-i-step).
In general in ruby when you increase (or decrease) the index manually its a sign you can use another build in function to encapsulate those actions.